openstack/datadog_checks/openstack/openstack.py

```diff
@@ -825,8 +844,7 @@ def check(self, instance):
             else:
                 self.warning("Couldn't get hypervisor to monitor for host: %s" % self.get_my_hostname())
 
-            if project:
-                self.get_stats_for_single_project(project)
+            self.get_stats_for_all_projects(projects)
```

Comment:
the `if project:` above seems to imply that `project` can be empty, is it still ok?

Reply:
That's right, I forgot to add a check to ensure that both a scoped `project` and a list of `projects` exist before running `get_stats_for_all_projects()` is ran